src/plugins/data_view_editor/public/components/data_view_editor_flyout_content.tsx

```diff
@@ -366,12 +364,7 @@ const IndexPatternEditorFlyoutContentComponent = ({
   );
 
   return (
-    <EmptyPrompts
-      onCancel={onCancel}
-      allSources={allSources}
-      loadSources={loadSources}
-      showEmptyPrompt={showEmptyPrompt}
-    >
+    <EmptyPrompts onCancel={onCancel} loadSources={loadSources}>
```

Comment:
looks like `allSources` is omitted from parameters now

Reply:
Yes, as we are using hasData service to check for existence of data, loadSources is not needed I believe.